statistics-dashboard-service/src/main/java/eu/europeana/statistics/dashboard/service/StatisticsService.java

```diff
@@ -33,6 +34,7 @@
 public class StatisticsService {
 
   private static final String STATISTICS_RESULT_ROOT_VALUE = "ALL_RECORDS";
+  private static final DecimalFormat PERCENTAGE_FORMAT = new DecimalFormat("0.00");
```

Comment:
Is this rounding up or down?
And what do we want?

Reply:
It is rounded up by default.
The value `0.00` when initializing DecimalFormat simply indicates how to represent the value, this ensures that the value of the percentage is presented up to 2 decimals, for example `60.087345` will be turned to `60.09`.